src/ActiveField.php

```diff
@@ -150,11 +149,13 @@ class ActiveField extends \yii\widgets\ActiveField implements Initiable
 
 
     /**
-     * {@inheritdoc}
+     * Constructor for activefield
      */
-    public function init(): void
+    public function __construct(array $config = [])
     {
-        AbstractContainer::configure($this, $this->createLayoutConfig());
+     	if (!empty($config)) {
+        	AbstractContainer::configure($this, $this->createLayoutConfig());
```

Comment:
What is $config purpose here? This looks off.

Reply:
https://github.com/yiisoft/yii-bootstrap4/commit/27744ee4d9cc5f52e0836a6e0fad7288752bca44

Reply:
@samdark could reverse the commit to the confirmation of @machour https://github.com/yiisoft/yii-bootstrap4/commit/c67c40ace499b6957ddeb5d5f7ea9ced84851b84, my pr it breaks all the tests.